neo4j-jdbc-bolt/src/test/java/org/neo4j/jdbc/bolt/BoltNeo4jStatementIT.java

```diff
@@ -305,4 +305,86 @@ public static void tearDown(){
 		statement.close();
 	}
 
+	@Test public void testCallProcedure() throws SQLException {
+		connection.setAutoCommit(true);
+
+		Statement statement = connection.createStatement();
+		statement.execute("CALL db.schema.visualization");
+
+		ResultSet resultSet = statement.getResultSet();
+		resultSet.next();
+		Array arrayNodes = resultSet.getArray("nodes");
+		Object[] nodesArray = (Object[]) arrayNodes.getArray();
+		assertEquals(0, nodesArray.length);
+		Array arrayRels = resultSet.getArray("relationships");
+		Object[] relsArray = (Object[]) arrayRels.getArray();
+		assertEquals(0, relsArray.length);
+
+		statement.close();
+	}
+
+	@Test public void testSimpleReturn() throws SQLException {
+		connection.setAutoCommit(true);
+
+		Statement statement = connection.createStatement();
```

Comment:
nit: why not use try-with-resources around statements?

Reply:
done